### ISSUE TITLE
Article Update: Safari keyboard navigation

### ISSUE THREAD

**Issue by erinoerin** [a11yproject/a11yproject.com#859]
Proposal for 3 Updates:

1. Rename article. This article covers keyboard navigation for all browsers on MacOS, not just Safari. Suggested name: Browser Keyboard Navigation in MacOSX. (Alternatively: make a new article for each browser)
2. Screenshots are now out of date. As of Catalina, there is only one option: "Use keyboard navigation to move focus between controls". Suggestions: Either replace the screenshot, or add an up-to-date version alongside older one.
3. Firefox. I'm not sure if/when checking the "Use keyboard navigation..." in Settings > Keyboard shortcuts became insufficient, but firefox will not tab to links (firefox 70, Catalina). To fix this, add the [accessibility.tabfocus config item](https://developer.mozilla.org/en-US/docs/Mozilla/Preferences/Preference_reference/accessibility.tabfocus) and set it to 7 in firefox about:config. I am _guessing_ that Firefox accessibility.tabfocus is 2 (Give focus to all form elements except text fields) even when "Use keyboard navigation..." is set, so links still wouldn't receive focus. 

I am happy to update the article

**Comment by ericwbailey** [comment]
Hi @erinoerin! We would love it if you could submit a PR to that article to update it. I've been personally holding off on Catalina, so I didn't realize the changes were so, uh, pervasive.

**Comment by danburzo** [comment]
Hi everyone, saw the article update in my RSS reader. I wanted to chip in with this [Bugzilla issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1587962) that tracks Firefox's macOS Catalina behavior. Seems it will be fixed in either FF 71, or 72, so manually altering `about:config` should not be needed in the (near) future.

**Comment by erinoerin** [comment]
Nice! It looks like tab behavior is working in Firefox Developer Edition 71.0b11, so that’s promising.

**Comment by ericwbailey** [comment]
Confirmed on my end. @erinoerin, could I trouble you to update the article when it hits stable? I think it'd be good to include the current Firefox workaround in the update, just in case.

**Comment by erinoerin** [comment]
@ericwbailey sure thing!
